ciri/api/api.c

```diff
@@ -28,7 +28,26 @@ static bool invalid_subtangle_status(iota_api_t const *const api, error_res_t **
     *error = error_res_new(API_INVALID_SUBTANGLE);
   }
 
-  return invalid;
+  char_buffer_set(res->app_name, CIRI_NAME);
+  char_buffer_set(res->app_version, CIRI_VERSION);
+  memcpy(res->latest_milestone, api->core->consensus.milestone_tracker.latest_milestone, FLEX_TRIT_SIZE_243);
+  res->latest_milestone_index = api->core->consensus.milestone_tracker.latest_milestone_index;
+  memcpy(res->latest_solid_subtangle_milestone, api->core->consensus.milestone_tracker.latest_solid_subtangle_milestone,
+         FLEX_TRIT_SIZE_243);
+  res->latest_solid_subtangle_milestone_index =
+      api->core->consensus.milestone_tracker.latest_solid_subtangle_milestone_index;
+  res->milestone_start_index = api->core->consensus.milestone_tracker.milestone_start_index;
+  rw_lock_handle_rdlock(&api->core->node.neighbors_lock);
+  res->neighbors = neighbors_count(api->core->node.neighbors);
+  rw_lock_handle_unlock(&api->core->node.neighbors_lock);
+  res->packets_queue_size = broadcaster_size(&api->core->node.broadcaster);
+  res->time = current_timestamp_ms();
+  res->tips = tips_cache_size(&api->core->node.tips);
+  res->transactions_to_request = requester_size(&api->core->node.transaction_requester);
+  features_set(res->features, &api->core->node.features);
+  memcpy(res->coordinator_address, api->core->consensus.conf.coordinator_address, FLEX_TRIT_SIZE_243);
+
+  return RC_OK;
```

Comment:
I think there is a failed conflict merging here, you are modifying the function `invalid_subtangle_status`